windows_containers/windows-containers-release-notes-3-x.adoc

```diff
@@ -17,6 +17,44 @@ The release notes for Red Hat OpenShift for Windows Containers tracks the develo
 
 You must have a subscription to receive support for the Red Hat WMCO. Deploying Windows container workloads in production clusters is not supported without a subscription. If you do not have a subscription, you can use the community WMCO, a distribution that lacks official support. Request support through the link:http://access.redhat.com/[Red Hat Customer Portal].
 
+[id="wmco-3-1-0"]
+== Release notes for Red Hat Windows Machine Config Operator 3.1.0
+
+Issued: 2021-09-21
+
+The WMCO 3.1.0 is now available with bug fixes and a new feature. The components of the WMCO were released in link:https://access.redhat.com/errata/RHBA-2021:3215[RHBA-2021:3215].
+
+[id="wmco-3-1-0-new-features"]
+=== New features
+
+[id="wmco-3-1-0-byoh"]
+==== Using Bring-Your-Own-Host (BYOH) Windows instances
+
+You can now add an existing Windows instance to an {product-title} cluster as a compute node. This requires creating a config map in the WMCO namespace.
+
+BYOH Windows instances are supported for the following platforms with installer-provisioned infrastructure:
+
+* Amazon Web Services (AWS)
+* Microsoft Azure
+* VMware vSphere
+
+BYOH Windows instances are supported for the following platforms with user-provisioned infrastructure when the `platform: none` field is set in the `install-config.yaml` file:
```

Comment:
```suggestion
BYOH Windows instances are supported in a User-Provisioned Infrastructure (UPI) with the `platform: none` field set in the `install-config.yaml` file, for the following platforms:
```

Reply:
Thanks, @jrvaldes. I updated this wording a bit :+1:. In our docs, we avoid using the "UPI" acronym, as it's more of an internal thing that we don't frequently broadcast out.